podlove/podlove.github.com
# Fix "Edit this page" links

When clicking on the "Edit this page" link on the docs website one will get a 404 error.

#### How to reproduce:

1. Go to the docs website: https://podlove.github.io/podlove-publisher/index.html
2. Click the "Edit this page" link
3. You will see a 404 error page of github.

#### How to fix

You can see, that the edit links guides to `https://github.com/podlove/podlove.github.com/edit/master/publisher/publisher/index.mdx`, but it should be:

```diff
- https://github.com/podlove/podlove.github.com/edit/master/publisher/publisher/index.mdx
+ https://github.com/podlove/podlove.github.com/edit/master/publisher/index.mdx
```

I'm not familiar with docusaurus, but I hope that this PR will fix this issue.
